frontend/src/add-citizen/add-citizen.vue

```diff
@@ -8,7 +8,7 @@
            no-close-on-backdrop
            no-close-on-esc
            class="m-0 p-0"
-           @shown="setupForm()">
+           @shown="setupForm()" v-dragged="onDrag">
```

Comment:
@denny351 i just put this in sprint2 since it's being trialed and they can see if they want it.